User/src/main/java/com/example/user/config/Config.java

```diff
@@ -0,0 +1,11 @@
+package com.example.user.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class Config {
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
Weź mi napisz do czego to wykorzystujesz bo nie wiem, a nie wydaje mi się żeby to było potrzebne